src/t_zset.c

```diff
@@ -2550,9 +2551,6 @@ static void zdiffAlgorithm2(zsetopsrc *src, long setnum, zset *dstzset, size_t *
         if (cardinality == 0) break;
     }
 
-    /* Resize dict if needed after removing multiple elements */
-    if (htNeedsResize(dstzset->dict)) dictResize(dstzset->dict);
-
```

Comment:
can you explain why did you remove this one and not do the pause / unpause here?

Reply:
Sorry. I missed `zdiffstore` before and I thought it was a temporary dict.